appveyor.yml

```diff
@@ -3,6 +3,9 @@ build: off
 cache:
   - 'C:\\tmp'
 environment:
+  global:
+    # Please, change to use token of PyCQA/pylint
+    COVERALLS_REPO_TOKEN: "R8noz2lfRpFXNPKQirwNHJyLl1wMIJnYF"
```

Comment:
Please, check follow oficial documentation from coveralls

<img width="779" alt="screen shot 2016-04-15 at 4 55 53 pm" src="https://cloud.githubusercontent.com/assets/6644187/14575981/14bc22a6-032b-11e6-8ec4-ecdd000e0867.png">

But if you don't want use this value for other CI... don't problem travis works!